subprojects/maven/src/main/java/org/gradle/api/publication/maven/internal/BasePomFilterContainer.java

```diff
@@ -75,6 +76,20 @@ public MavenPom pom(String name, Closure configureClosure) {
         return ConfigureUtil.configure(configureClosure, pom(name));
     }
```

Comment:
That was intentional to avoid creating one more object per invocation of those methods since there's very little duplication left.